cards/src/main/resources/cards/custom/group0/other_death_knight/oldgods/permanent_puzzle_box.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Puzzle Box",
+  "baseManaCost": 4,
+  "type": "MINION",
+  "heroClass": "SPIRIT",
+  "baseAttack": 0,
+  "baseHp": 1,
+  "rarity": "EPIC",
+  "description": "Remove the top 4 cards of your deck. In 4 turns, add them to your hand.",
```

Comment:
`"Permanent. Battlecry: Remove the top 4 cards of your deck. In 4 turns, add them to your hand and remove this"`